src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -2500,28 +2500,15 @@ void MessageHelper::SendOnDataStreaming(
   app_mngr.GetRPCService().ManageHMICommand(notification);
 }
 
-bool MessageHelper::SendStopAudioPathThru(ApplicationManager& app_mngr) {
-  LOG4CXX_INFO(logger_, "MessageHelper::SendAudioStopAudioPathThru");
+smart_objects::SmartObjectSPtr MessageHelper::CreateMessageWithFunctionID(
+    ApplicationManager& app_mngr, hmi_apis::FunctionID::eType function_id) {
+  LOG4CXX_INFO(logger_, "MessageHelper::CreateMessageWithFunctionID");
```

Comment:
TRACE

Reply:
Done in da940c2e0e2ea1d863b840c2a9204159e0d6bfc9